java/client/src/main/java/glide/api/commands/ConnectionManagementClusterCommands.java

```diff
@@ -48,4 +49,80 @@ public interface ConnectionManagementClusterCommands {
      * @return <code>String</code> with a copy of the argument <code>message</code>.
      */
     CompletableFuture<String> ping(String message, Route route);
+
+    /**
+     * Gets the current connection id.<br>
+     * The command will be routed a random node.
+     *
+     * @see <a href="https://redis.io/commands/client-id/">redis.io</a> for details.
+     * @return The id of the client.
+     * @example
+     *     <pre>
```

Comment:
Yes
I agree it looks weird.
Linter always moves `pre` to the next line, but to keep it as a part of `@example` block - adds indentation. Next lines are not indented, because they are pre-formatted.